lib/collection/header.js

```diff
@@ -44,12 +44,7 @@ _.inherit((
      */
     Header = function PostmanHeader (options, name) {
         if (_.isString(options)) {
-            options = Header.parseSingle(options);
-        }
-
-        if (options && _.isString(name)) {
-            options.value = options.key;
-            options.key = name;
+            options = _.isString(name) ? { key: name, value: options } : Header.parseSingle(options);
```

Comment:
while running the tests, the parameter `options` is passed as `Mon, 25 Jul 2016 13:11:41 GMT`  and `name` as `Date`. `parseSingle` breaks for date values like `Mon, 25 Jul 2016 13:11:41 GMT` returning the value as instead `Mon, 25 Jul 2016 13`